apps/src/netsim/NetSimRouterNode.js

```diff
@@ -541,14 +611,13 @@ NetSimRouterNode.prototype.routeMessage_ = function (message, myWires) {
   var toAddress = message.payload.toAddress;
   if (toAddress === undefined) {
     //Malformed packet? Throw it away
-    logger.warn("Router encountered a malformed packet: " +
-    JSON.stringify(message.payload));
+    this.log("Blocked malformed packet: " + message.payload);
     return;
   }
 
   if (toAddress === 0) {
```

Comment:
I believe this logic can be removed, and packets addressed to 0 can be treated like any other incorrectly addressed packet. This is based on the assumption that packets addressed to the router are not part of the spec.